packages/react-dom-bindings/src/server/ReactDOMServerFormatConfig.js

```diff
@@ -114,6 +115,7 @@ export type ResponseState = {
   sentCompleteBoundaryFunction: boolean,
   sentClientRenderFunction: boolean,
   sentStyleInsertionFunction: boolean, // We allow the legacy renderer to extend this object.
+  sentExternalRuntime: boolean,
```

Comment:
Ahh that makes sense, we plan to send the runtime lazily 

For readability, would it be better to refactor these instruction writing states into separate structs? I think #25437 adds this already, but just curious in terms of style

Reply:
oh yeah I meant to remove that before committing, my bad. We can address in your PR.